Sources/LowEnergyCommandParameter.swift

```diff
@@ -223,45 +223,77 @@ public extension LowEnergyCommand {
         public static let length = 2 + 2 + 1 + 1 + 1 + 6 + 1 + 1
         
         /// Interval for non-directed advertising.
-        public var interval: (minimum: UInt16, maximum: UInt16)
+        ///
+        /// - Note: The `Advertising_Interval_Min` and `Advertising_Interval_Max` should not be the same value
+        /// to enable the Controller to determine the best advertising interval given other activities.
+        ///
+        /// For directed advertising, when Advertising_Type is `0x01` (ADV_DIRECT_IND),
+        /// the `Advertising_Interval_Min` and `Advertising_Interval_Max` parameters are not used and shall be ignored.
+        ///
+        /// The `Advertising_Interval_Min` and `Advertising_Interval_Max` shall not be set to less than
+        /// `0x00A0` (100 ms) if the Advertising_Type is set to `0x02` (ADV_SCAN_IND) or `0x03` (ADV_NONCONN_IND).
+        public var interval: (min: AdvertisingInterval, max: AdvertisingInterval)
         
+        /// Ssed to determine the packet type that is used for advertising when advertising is enabled.
         public var advertisingType: AdvertisingType
         
-        public var addressType: (own: LowEnergyAddressType, direct: LowEnergyAddressType)
+        /// Determines if the advertising packets are identified with the Public Device Address of the device,
+        /// or a Random Device Address as written by the `LE_Set_Random_Address` command.
+        public var ownAddressType: LowEnergyAddressType
+        
+        /// Address type
+        public var directAddresssType: LowEnergyAddressType // Direct_Address_Type
         
         /// Public Device Address or Random Device Address of the device to be connected.
         public var directAddress: Address
         
-        public var channelMap: ChannelMap
+        public var channelMap: BitMaskOptionSet<ChannelMap>
         
         public var filterPolicy: FilterPolicy
         
-        public init(interval: (minimum: UInt16, maximum: UInt16) = (0x0800, 0x0800),
+        public init(interval: (min: AdvertisingInterval, max: AdvertisingInterval) = (.default, .default),
```

Comment:
/// - Note: The `Advertising_Interval_Min` and `Advertising_Interval_Max` should not be the same value
/// to enable the Controller to determine the best advertising interval given other activities.
The default value are the same and the notes states that the should be the same